posta/utils.py

```diff
@@ -9,4 +10,4 @@ def imposta_destinatari_e_scrivi_messaggio(request, qs_destinatari=Persona.objec
     qs_destinatari = Persona.objects.filter(pk__in=qs_destinatari.values_list('id', flat=True))
     request.session["messaggio_destinatari"] = qs_destinatari
     request.session["messaggio_destinatari_timestamp"] = datetime.now()
-    return redirect("/posta/scrivi/")
+    return redirect("{}?da_elenco=1".format(reverse('posta-scrivi')))
```

Comment:
Sono completamente d'accordo che il `reverse` sia il metodo corretto per fare questa roba. Allo stesso tempo e' in disaccordo con tutto il resto del codice e ho paura che la complessita' di scrivere un URL in questo modo, semplicemente, non ne valga la pena.